src/pages/Main/MainCarousel.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import ImgCom from './ImgCom';
+
+class MainCarousel extends Component {
+  constructor() {
+    super();
+    this.state = {
+      firstImg: 0,
+      sliderArr: [],
+      info: {},
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/visualData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          sliderArr: data,
+        });
+      });
+  }
+
+  slideLeft = () => {
+    const { firstImg } = this.state;
+    firstImg === 0
+      ? this.setState({ firstImg: 0 })
+      : this.setState({ firstImg: firstImg + 100 });
```

Comment:
- setState 문을 반복할 필요 없이, 달라지는 부분에 대해서만 삼항 연산자를 사용해 주시면 더 좋을 것 같습니다~
```suggestion
    this.setState({
      firstImg: firstImg === 0 ? 0 : firstImg + 100,
    })
```